assets/css/style.scss

```diff
@@ -38,7 +38,7 @@
 .editor-styles-wrapper .wc-block-grid__product-title,
 .wc-block-grid__product-title {
 	font-family: inherit;
-	line-height: 1.2em;
+	line-height: 1.2;
```

Comment:
Is this change also intentional? The meaning is slightly different.

Reply:
It was marked as an error by updated stylelint. These mean the same, unless I missed something. em is relative unit to current font size, and unitless line-height is ratio of current font size. In context of line-height these produce the same value.

Reply:
It's just slightly different, and generally recommended to go without `em` so I'd agree with stylelint, but it might have changed stuff.

The difference is the following:

Imagine `.wc-block-grid__product-title` has `font-size: 10px`. Now imagine that we have a child element of `.wc-block-grid__product-title` called `.subtitle`, which has `font-size: 5px`.

Now, if `line-height` on `.wc-block-grid__product-title` is set to `1.2em`, the computed `line-height` on `.subtitle` will be `12px` (generally not what one wants). Otherwise, if set to `1.2`, it will be `6px`. In both cases, `line-height` on `.wc-block-grid__product-title` will be `12px`.

Does it make sense?

Again, generally it's more correct to go with the fix you made, I was just wandering if you tested for any differences.

Reply:
Ah, I see what you mean. Spit out a quick https://codepen.io/tomasztunik/pen/yLpbjgw to make it stuck for longer :) In this case this shouldn't be an issue as the title block doesn't have a child. Though this gets me thinking if this linter setting that comes from WordPress recommended is not going to bite us in the future somehow. In theory it shouldn't and if it reports error in an unexpected way it will resurface and we'd be able to override it with `.stylelintrc.json`. I think for now we should go with the default - what do you think?

Reply:
I'm totally fine with using this, though I think there were more changes like that to line heights that might have children. So perhaps just do a quick before/after check of these elements in context?

I'll approve to not block you in the meantime.

Reply:
confirmed that all of them should be good and were on content elements not on containers.